### PR TITLE
Euler Problems

### PR DESCRIPTION
**Pull request by rodriguez-maria**
I have solved 5 problems from the Euler Project website.

### CODE REVIEW

File: src/dev/MultiplesThreeAndFive.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+If we list all the natural numbers below 10 that are multiples of 3 or 5, 
+we get 3, 5, 6 and 9. The sum of these multiples is 23.
+Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+
+package dev;
+
+public class MultiplesThreeAndFive {
```
Comment (rfaisal) [inline src/dev/MultiplesThreeAndFive.java]: MultiplesThreeOrFive

File: src/dev/SmallestMultiple.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+package dev;
+
+public class SmallestMultiple {
+	
+	public static long findSmallestMultiple(int max)
+	{
+		long smallestNum = 0;
+		boolean isTheSmallest = false;
+		long currentNum = 2520;
```
Comment (rfaisal) [inline src/dev/SmallestMultiple.java]: where did you get this number?

File: src/dev/SumSquareDifference.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		for(int i = 1; i <= n; i++)
+		{
+			sumOfSquares += Math.pow(i, 2);
+		}
+		
+		for(int i = 1; i <= n; i++)
+		{
+			squareOfSum += i;
+		}
+
```
Comment (rfaisal) [inline src/dev/SumSquareDifference.java]: both of the loops could be done in one

File: src/dev/SummationOfPrimes.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+			if(n == 2 || prime(i))
+			{
+				sumOfPrimes += i;
+			}
+		}
+		
+		return sumOfPrimes;
+	}
+		
+		private static boolean prime(long n)
```
Comment (rfaisal) [inline src/dev/SummationOfPrimes.java]: should rename to isPrime

File: src/dev/MultiplesThreeAndFive.java (hunk 1 above)
Comment (rodriguez-maria) [reply src/dev/MultiplesThreeAndFive.java]: Done

File: src/dev/SmallestMultiple.java (hunk 2 above)
Comment (rodriguez-maria) [reply src/dev/SmallestMultiple.java]: solved

File: src/dev/SumSquareDifference.java (hunk 3 above)
Comment (rodriguez-maria) [reply src/dev/SumSquareDifference.java]: Excellent thank you =)

File: src/dev/SummationOfPrimes.java (hunk 4 above)
Comment (rodriguez-maria) [reply src/dev/SummationOfPrimes.java]: Thank you!!!

**Pull request merged**